components/script/dom/servohtmlparser.rs

```diff
@@ -522,7 +522,7 @@ struct Tracer {
 impl tree_builder::Tracer for Tracer {
     type Handle = JS<Node>;
     #[allow(unrooted_must_root)]
-    fn trace_handle(&self, node: JS<Node>) {
```

Comment:
Could `Handle` be `Node` in this case to prevent extra indirection?

r+ either way